### PR TITLE
Add session expiry end point

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
This PR adds the session-expiry end point, which returns expires_at in response object with the application/json mimetype

### How to review 
Launch a questionnaire and then visit /session-expiry to check it works and changes with time

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: app/routes/session.py [hunk 1]
```diff
@@ -121,6 +121,12 @@ def get_session_expired():
     return render_template("errors/401")
 
 
+@session_blueprint.route("/session-expiry", methods=["GET"])
```
Comment (MebinAbraham) [inline app/routes/session.py:124]: Was there a reason this wasn't `session/expiry`?. That is what was intended by the card. I think it should live off the session route, I think the same for the existing endpoint in this file, but that don't need to change here.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

File: app/routes/session.py (hunk 1 above)
Comment (LJBabbage) [reply app/routes/session.py]: You're 18 days late on this request and what are you doing going through old stories :laughing:. It was just aligned with the other endpoints. Being only a spike and already being worked on by Matt I don't think we should change it at the moment, but can before it gets merged

Comment (MebinAbraham) [reply app/routes/session.py]: PR still open... would have asked if I wasn't on holiday. Doesn't need to change while Matt is developing, just a simple question to see if there was a reason it doesn't match the card or just an oversight, maybe a covo that happened that I wasn't aware off... no need for the snarky comment 🤦

Comment (LJBabbage) [reply app/routes/session.py]: Not snarky, I used the laughing emoji.

**Pull request closed without merging**